play/src/front/Components/ActionBar/ActionBar.svelte

```diff
@@ -782,34 +782,28 @@
                         <img draggable="false" src={menuImg} style="padding: 2px" alt={$LL.menu.icon.open.menu()} />
                     </button>
                 </div>
-                {#if $mapEditorActivated}
-                    <!-- svelte-ignore a11y-click-events-have-key-events -->
-                    <div
-                        on:dragstart|preventDefault={noDrag}
-                        on:click={toggleMapEditorMode}
-                        class="bottom-action-button"
+                <!-- svelte-ignore a11y-click-events-have-key-events -->
+                <div on:dragstart|preventDefault={noDrag} on:click={toggleMapEditorMode} class="bottom-action-button">
+                    {#if isMobile}
+                        <Tooltip text={$LL.actionbar.mapEditorMobileLocked()} />
```

Comment:
The map editor is locked in mobile mode (hence the special label) but.... the exploration mode should probably be allowed (it is really useful in mobile, isn't it?)

Reply:
Today it's use with the map editor tool. So we need to create specific content for mobile.